### ISSUE TITLE
Running on example NGS data

### ISSUE THREAD

**Issue by yangyulin0809** [jyyu/ScreenBEAM#5]
Hi jyyu,
Does the example NGS data (NGS.example.tsv) contains the raw read counts without doing any normalization?
Besides, I met some warnings when I run ScreenBEAM on NGS data like following. Should I just ignore them?
![image](https://cloud.githubusercontent.com/assets/16235664/25367811/ccd9c496-29aa-11e7-90ce-171ecf2feba2.png)

**Comment by DarioS** [comment]
The documentation of this R package is of poor quality and the package doesn't pass minimum standards to be accepted into CRAN (i.e. Lots of functions in utilities.R missing Rd help files), let alone Bioconductor (i.e. No vignette). It's going to be a struggle to understand and use it effectively. No biological study published in the past 2 years has used it.

**Comment by jyyu** [comment]
It can take both. There is a do.normalization para, by default TRUE.

**Comment by jyyu** [comment]
@DarioS We believe we put sufficient info for users even R beginners to run the analysis. I have been told by tons of users that they can run it without any problem. The utility or internal functions are really not for general users.

We didn't advertise this method until recently I started my own lab and presented in conferences. Many researchers started using it since then. As far as I know, many projects which used our method are in validation stage, which might take time to get published.
 
We didn't put the first version on CRAN and Bioconductor but on GitHub, just for researchers who might be interested. I wish I had much time to polish the library. Since I just started my lab, we hope to release a new version with much more features as soon as my staff scientists and soft engineers are in position. Will incorporate your suggestions as much as possible in the new release.

**Comment by yangyulin0809** [comment]
Hi Jyyu, I believe ScreenBEAM can be very useful to me. Actually, I also tried to install MAGeCK-VISPR on my server, but I met so many troubles that I gave it up finally! ScreenBEAM is very easy to install for me!
Much appreciate your great work!

**Comment by DarioS** [comment]
You can avoid installation headaches by using [CRISPRAnalyzeR](http://crispr-analyzer.dkfz.de/). It's a web application that will run both ScreenBEAM and MAGeCK on your dataset. You can then explore the overlaps of the genes found between competing methods.

**Comment by jyyu** [comment]
Yes, CRISPRAnalyzeR is a nice tool, which runs ScrennBEAM and others. Thanks for sharing.
